queries.sql

```diff
@@ -7,4 +7,11 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *  FROM animals WHERE neutered;
 SELECT *  FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT *  FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT *  FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT COUNT(*) FROM animals;
+SELECT * FROM animals WHERE escape_attempts = 0;
```

Comment:
- [ ] This one should count the animals that have tried to escape, not list them.  Example:

`   SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;`

Kindly correct that one. 💪

Reply:
Thank you for that observation. I have successfully worked on that